spec/requests/notifications_controller_spec.rb

```diff
@@ -288,6 +288,80 @@ def delete_notification(resp_code, matcher)
           delete_notification(403, :to)
         end
       end
+
+      describe '#mark_read' do
+        context "when targeting a notification by id" do
+          it 'can mark a notification as read' do
+            expect {
+              put "/notifications/mark-read.json", params: { id: notification.id }
+              expect(response.status).to eq(200)
+              notification.reload
+            }.to change { notification.read }.from(false).to(true)
+          end
+
+          it "doesn't mark a notification of another user as read" do
+            notification.update!(user_id: Fabricate(:user).id, read: false)
+            expect {
+              put "/notifications/mark-read.json", params: { id: notification.id }
+              notification.reload
+            }.not_to change { notification.read }
+          end
+        end
+
+        context "when targeting multiple notifications by type" do
```

Comment:
```suggestion
        context "when targeting notifications by type" do
```

I think we can leave out multiple here.